src/creatures/players/player.cpp

```diff
@@ -155,42 +158,30 @@ std::string Player::getDescription(int32_t lookDistance) const {
 		}
 		s << '.';
 
-		std::string pronoun;
-		if (sex == PLAYERSEX_FEMALE) {
-			pronoun = " She";
-		} else {
-			pronoun = " He";
-		}
-		s << pronoun;
+		s << " " << pronoun;
 
 		if (group->access) {
-			s << " is " << group->name << '.';
+			s << " " << getSubjectVerb() << " " << group->name << '.';
 		} else if (vocation->getId() != VOCATION_NONE) {
-			s << " is " << vocation->getVocDescription() << '.';
+			s << " " << getSubjectVerb() << " " << vocation->getVocDescription() << '.';
 		} else {
 			s << " has no vocation.";
 		}
 
 		if (loyaltyTitle.length() != 0) {
-			if (sex == PLAYERSEX_FEMALE) {
-				s << " She is a " << loyaltyTitle << ".";
-			} else {
-				s << " He is a " << loyaltyTitle << ".";
+			std::string article = "a";
+			if (loyaltyTitle[0] == 'A' || loyaltyTitle[0] == 'E' || loyaltyTitle[0] == 'I' || loyaltyTitle[0] == 'O' || loyaltyTitle[0] == 'U') {
+				article = "an";
 			}
-		}
-
-		if (isVip()) {
-			s << pronoun << " is VIP.";
+			s << " " << pronoun << " " << getSubjectVerb() << " " << article << " " << loyaltyTitle << ".";
```

Comment:
are you sure? removing info about vip ?

Reply:
added it back, had been removed incidentally since I didn't actually have this on mine